code/datums/outfit.dm

```diff
@@ -238,4 +238,13 @@
 		if(imptype)
 			implants += imptype
 	accessory = text2path(outfit_data["accessory"])
-	return TRUE
\ No newline at end of file
+	return TRUE
+
+
+/datum/outfit/mulligan
+	name = "a Mulligan"
```

Comment:
name it "a Mulligan (Naked)" instead, admins dont have the highest IQ when it comes to using admin tools